Newsletters: add 51 (2019-06-19)

harding opened the pull request:
Todo:

- [x] @jnewbery add Breaking Bitcoin conf report

moneyball commented on _includes/specials/bech32/14-security.md:
```diff
@@ -0,0 +1,117 @@
+of the parties involved in generation of the address and so may be able
+to manipulate what address is finally chosen.  For example, Bob sends
+his pubkey to Mallory expecting that Mallory will send her pubkey back.
+The he expects they'll each put the pubkeys into a multisig script
```
"The he" typo

moneyball commented on _posts/en/newsletters/2019-06-19-newsletter.md:
```diff
@@ -90,6 +91,100 @@ projects.
+
+- [**Extracting Seeds from Hardware Wallets**][bb guillemet video] - Ledger CSO
+  [Charles Guillemet][] gave an eye-opening talk about security issues with
+  several hardware wallet on the market. He spoke about previously-revealed
```
typo: wallets

jnewbery replied:
fixed

jnewbery commented on _posts/en/newsletters/2019-06-19-newsletter.md:
I haven't looked into this BIP or mailing list post. Do you think it's generally interesting for our readers? I haven't heard anyone talking about it before.

harding replied:
A redundant question---I only put things in the newsletter if I think they're generally interesting for the readers.  Of course, I sometimes have strange tastes, so it's good to question them.  :-)  In this case I'm adding an extra sentence describing a possible application with a link to details so that the motivation is clearer.

For discussion, unfortunately most of that seems to have occurred on the PR rather than the mailing list where it would've been more appropriate; still, if you look at the PR you'll find it was well discussed---102 comments from many well-known contributors.

jnewbery replied:
Thanks! I should have phrased this as "I'm not convinced that this is generally interesting. Can you give some justification please?", which you've done. Looks good to me now, except with the observation that you've repeated 'proposed' in the sentence. Perhaps reword as:

"This idea was first proposed to the Bitcoin-Dev mailing list over two years ago. A suggested use-case is identifying which transactions contain information useful to third-party applications, such as timestamped decentralized identity references." or similar.

jnewbery commented:
Pushed a commit to address a few minor typos/grammar errors.

ACK b85871bbf3b7075d559e5fc7d783ca69b61f775a

harding commented:
Pushed a few commits addressing feedback (thanks!) and a commit that makes some small edits to @jnewbery's excellent BB writeup (please feel to revert any of those changes).

jnewbery commented:
Thanks for the edits @harding!

ACK 2949e54fb440cccbd78c1eb61b5748690a474965

harding commented:
Pushed a remove-only edit that gets rid of the bit about what LND branch to use with the watchtower tutorial as the tutorial has been updated to use LND master.

bitschmidty commented:
tACK 2372c66652f7393b8a1a9c91aa3602d129d3f958

bitschmidty commented:
squashed and force pushed with a minor typo fix

bitschmidty commented:
We are merged! Thanks for @harding and @jnewbery for authoring and @moneyball on the reviewing!
